Plugin.pm

```diff
@@ -628,6 +635,164 @@ sub getFeatured {
 	});
 }
 
+sub getHome {
+	my ( $client, $cb, $args, $params ) = @_;
+
+	getAPIHandler($client)->page(sub {
+		my $modules = shift;
+
+		my $items = [ map {
+			{
+				name => $_->{title},
+				type => 'link',
+				url => $_->{type} eq 'HIGHLIGHT_MODULE' ? \&getHighlights : \&getModule,
+				passthrough => [ { module => $_ } ],
+			}
+		} @$modules ];
+
+		$cb->( {
+			items => $items
+		} );
+	}, 'pages/home' );
+}
+
+sub getHighlights {
+	my ( $client, $cb, $args, $params ) = @_;
+
+	my $module = $params->{module};
+	my $items = [];
+
+	foreach my $entry (@{$module->{highlights}}) {
+		my $item;
+		my $title = $entry->{title};
+
+		$entry = $entry->{item};
+
+		# formats are not exactly consistent with the rest of the API. Small changes
+		# but requires a few exceptions. We could change getImageUrl and typeOfItem...
+		if ($entry->{type} eq 'ALBUM') {
+			$item = _renderAlbum($entry->{item}, { addArtistToTitle => 1 });
+		}
+		elsif ($entry->{type} eq 'PLAYLIST') {
+			$item = _renderPlaylist($entry->{item});
+			# image can't be 1280x1280...
+			my $image = $entry->{item}->{squareImage} =~ s/-/\//gr;
+			$item->{image} = IURL . $image . "/640x640.jpg";
+		}
+		elsif ($entry->{type} =~ /MIX/) {
+			$item = _renderMix($client, $entry->{item});
+			# image uses a different key (S/M/L) than current MIX
+			$item->{image} = $entry->{item}->{images}->{MEDIUM}->{url} || $entry->{item}->{images}->{SMALL}->{url} || $entry->{item}->{images}->{LARGE}->{url};;
+		}
+		elsif ($entry->{type} eq 'TRACK') {
+			# I don't think we should cache it in Async b/c that means a lot of knowledge of the format
+			my ($track) = @{Plugins::TIDAL::API->cacheTrackMetadata([ $entry->{item} ])};
+			$item = _renderTrack($track, { addArtistToTitle => 1 });
+		}
+		else {
+			$log->warn("unknow hightlight type: $entry->{type}");
+		}
+
+		$item->{name} = "$title: $item->{name}" unless $entry->{type} eq 'MIX';
+		push @$items, $item;
+	}
+
+	$cb->({
+		image => 'plugins/TIDAL/html/personal.png',
```

Comment:
I never got to see this icon. Where would I find the "Highlights"? From the above `getHome()` I would expect this would somehow be mixed in with the "Modules"? In that case we should have icons for both of them, or none. Modules currently don't have icons. But as mentioned elsewhere: that's ok, if there would be all of the same icons anyway.

Reply:
You should see it as soon as you hit "For you" item.

Reply:
I have several "Mix|Radio|New.. for you", but no "For you" alone. I navigated down the whole tree....

Reply:
The mysteries of Canada vs Switzerland I guess ... (or English vs German)